Backpack-SwiftUI/Price/Classes/BPKPrice.swift

```diff
@@ -32,23 +32,29 @@ public struct BPKPrice: View {
     private let leadingText: String?
     private let previousPrice: String?
     private let trailingText: String?
+    private let icon: BPKIcon?
     private let alignment: Alignment
     private let size: Size
+    private let accessibilityLabel: String?
     
     public init(
         price: String,
         leadingText: String? = nil,
         previousPrice: String? = nil,
         trailingText: String? = nil,
+        icon: BPKIcon? = nil,
         alignment: Alignment = .leading,
-        size: Size
+        size: Size,
+        accessibilityLabel: String? = nil
```

Comment:
Can you make the icon and accessibility label be in a tuple? So that if someone sets an icon, they also have to set a a11ylabel